credits.txt

```diff
@@ -61,3 +66,117 @@ Beta Testers
   Sean Fahey
   Mauricio Gomes
   Iaz Poolar
+
+Special thanks to MCOfficer for
+creating, managing, and hosting
+the nightly builds that makes
+so much of this work possible.
+
+Proof Readers & Editors
+  TheMarksman
+
+Mission & Story Creators
+  Amazinite
+  Zitchas
+
+Thank you to all the following
+people on GitHub who have
+contributed to improving Endless
+Sky:
```

Comment:
The home page of the repo says 151 contributors, but the contributors page only shows 100 (and here we have listed 96, the four developers mentioned as being excluded). Anyone know how to get the remaining 51 (possibly without needing to go through every PR)?

Reply:
Thanks! I've updated the list.